### DISCUSSION TITLE
forum-session-SciELO-en

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/accessibility2024#134] [General]
# forum-session-SciELO-en

This groundbreaking session on accessibility in the SciELO community is also a first for arXiv: our first non-English event. It will be held only in Spanish and Portuguese with simultaneous translation between those two languages (not English). If you do not speak Portuguese or Spanish you can help by spreading the word to your colleagues!

https://accessibility2024.arxiv.org/forum-session-SciELO-en

**Comment by coder4cbus** [comment]
How about incorporation of AI to allow for in real time translation to English? ;)

**Reply by car222222** [reply]
Sure, and do not forget time-shifting too, so that everyone can participate!

**Comment by coreballcore** [comment]
SciELO Preprints stands as a reliable preprints server adhering to international standards. The responsibility for the communications lies with the authors themselves, and the [dissemination](https://core-ball.org) of preprints is to be accompanied by a notice stating that they have not yet been peer reviewed.
